cla/CLA_BOT_PRIVACY_POLICY.md

```diff
@@ -0,0 +1,65 @@
+### Contributor License Agreement with Mercedes-Benz Tech Innovation GmbH Wilhelm-Runge-Straße 11 89081 Ulm
```

Comment:
Could you please rename this file? CLA_BOT_PRIVACY_POLICY implies, that we are talking about a privacy policy, but the document contains the CLA of Mercedes-Benz Tech Innovation. This should be reflected in the filename.